### ISSUE TITLE
Detection of Python version

### ISSUE THREAD

**Issue by TD-er** [tvgrabbers/tvgrabnlpy#53]
The latest grabber (2.2.8) I tested this morning, it exits on the following:
```
# check Python version
if sys.version_info[:3] < (2,7,9):
    sys.stderr.write("tv_grab_nl_py requires Pyton 2.7 or higher\n")
    sys.exit(2)
```
The error should also mention the latest digit.
My Mythbuntu has 2.7.4 (not entirely sure) and the Ubuntu 14.04 here has 2.7.6
So is it really needed to have at least such a new version, or is a work-around possible?

**Comment by hikavdh** [comment]
Yes it is needed for https support. They backported it from 3 among with some other stuff. And the program downloads basic data from github. Sorry for forgetting to update the message!

**Comment by jteeuw** [comment]
I have the same problem : tv_grab_nl_py requires Pyton 2.7 or higher

if i do a python -V i get the following output:
Python 2.7.3

**Comment by hikavdh** [comment]
You do need 2.7.9 or higher for the https support, else unexpected things can happen on the datafile download from github. This datafile contains all up-to-date info on the source merging etc. The program won't work without this data.
